### DISCUSSION TITLE
Working or not?

### DISCUSSION THREAD

**Discussion by phsamson** [andsfonseca/dota2-rpc-client#57] [Q&A]
Hello, how do I know if this is working? Is there any indication for me to know if it is working on my discord?

**Comment by andsfonseca** [comment]
Hi @phsamson.

It's not trivial to check, but I'll try to show some steps if the app is installed and working.

The simplest way.

1. Open discord (To check Rich Presence)
2. Install the application or start it if it is already installed.
3. Open Dota, then open the demo or play some game.
4. If everything is ok, you should see the status of your chosen hero.

If it is not appearing, there are some steps to check.

The application installation mode creates a service called `dota-rpc-client.exe` on the system. You can check it through the task manager or by using the "services.msc" command.

If the service doesn't appear, the installation failed. Just try installing again and if it doesn't work, post the log here and we'll take a look.

If installed, check if it starts. There are cases where the app doesn't find discord and stops execution.

If it doesn't want to start, you can start the application like a normal application. It will open a terminal, it's not ideal, but it's good to make sure there are no problems
